DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/_DelegateCourseInfoCard.cshtml

```diff
@@ -105,6 +105,15 @@
           <dd class="nhsuk-summary-list__actions"></dd>
         </div>
 
+        <div class="nhsuk-summary-list__row">

```

Comment:
In the wireframe this is a label rather than a table row. Assuming this is how they want it, please could you update the Jira ticket to reflect this change so as not to be flagged as an issue during testing.

Reply:
Updated Jira as suggested